l10n_ec_account_edi/models/account_edi_document.py

```diff
@@ -680,8 +683,43 @@ def _l10n_ec_get_info_debit_note(self):
             "importeTotal": self._l10n_ec_number_format(amount_total, 6),
             "pagos": debit_note._l10n_ec_get_payment_data(),
             "detalles": self._l10n_ec_header_get_document_lines_edi_data(taxes_data),
-            "infoAdicional": self._l10n_ec_get_info_aditional(),
+            "infoAdicional": self._l10n_ec_get_info_additional(),
         }
 
         debit_note_dict.update(self._l10n_ec_get_info_tributaria(debit_note))
         return debit_note_dict
+
+    @api.model
+    def send_mail_to_partners(self):
+        all_companies = self.env["res.company"].search(
+            [
+                ("partner_id.country_id.code", "=", "EC"),
+                ("l10n_ec_type_environment", "=", "production"),
+            ]
+        )
+        for company in all_companies:
+            self.with_company(company).send_mail_to_partner()
+
+    @api.model
+    def send_mail_to_partner(self):
```

Comment:
@cormaza 
I rename method name to `l10n_ec_send_mail_to_partner` but method is keep into `account.edi.document `because is not exclusive of `account.move`, with module [l10n_delivery_note](https://github.com/OCA/l10n-ecuador/pull/40) is need sent mail and i  will think inherit this function from `account.edi.document` instead of `account.move`.

Bonnus: I changed domain `("edi_document_ids.state", "=", "sent")` to  `("l10n_ec_authorization_date", "!=", False)` because is more specific of Ecuadorian documents with edit_format `l10n_ec_format_sri`
https://github.com/Odoo-EC/l10n-ecuador/blob/9d97b3d50530e73a0838d1920ddd33fc01b37a4f/l10n_ec_account_edi/models/account_move.py#L72-L77

Please update your review